src/wattsi.pas

```diff
@@ -504,9 +505,7 @@    TCrossReferences = record
 
                   Scratch := Default(Rope);
                   Scratch.Append('#');
-                  ExtractedData := Element.GetAttribute('id');
-                  Assert(not ExtractedData.IsEmpty);
```

Comment:
So should we add more asserts then? These were the only two places that did that.